renovate.json

```diff
@@ -9,7 +9,7 @@
     ":automergeDisabled",
     ":autodetectPinVersions"
   ],
-
+  "ignoreDeps": ["com.google.apis:google-api-services-dataflow"],
```

Comment:
Probably should add a comment here that we will follow the readme, @anguillanneuf mentioned, and we need to figure out how to update this when needed.